src/components/gui/query-explain.tsx

```diff
@@ -0,0 +1,119 @@
+import { DatabaseResultSet } from "@/drivers/base-driver";
+import { useMemo } from "react";
+import { z } from "zod";
+
+export interface QueryExplanationProps {
+  data: DatabaseResultSet;
+}
+
+export interface ExplainRow {
+  id: number;
+  parent: number;
+  notused: number;
+  detail: string;
+}
+
+export type ExplainRowWithChildren = ExplainRow & {
+  children?: ExplainRowWithChildren[];
+};
```

Comment:
Everything looks solid. But since we don't use it somewhere else, there is no reason to export it out

Reply:
Pushed an update. I cleaned up some stuffs, remove unnecessary exports and use consistent naming